clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractMembershipManager.java

```diff
@@ -224,6 +227,12 @@ public abstract class AbstractMembershipManager<R extends AbstractResponse> impl
         this.clientTelemetryReporter = clientTelemetryReporter;
         this.time = time;
         this.metricsManager = metricsManager;
+
+        // Update the group member ID label in the client telemetry reporter.
+        // According to KIP-1082, the consumer will generate the member ID as the incarnation ID of the process.
+        // Therefore, we can update the group member ID during initialization.
+        clientTelemetryReporter.ifPresent(reporter -> reporter.updateMetricsLabels(
```

Comment:
we don't need to pass `clientTelemetryReporter` to `AbstractMembershipManager` as we can call `updateMetricsLabels` after initializing `AbstractMembershipManager`, right? The member is fixed now